src/Common/HashTable/FixedHashTable.h

```diff
@@ -294,36 +296,28 @@ class FixedHashTable : private boost::noncopyable, protected Allocator, protecte
 
     const_iterator begin() const
     {
-        if (!buf)
+        if (!buf && min > max)
```

Comment:
maybe it should be `!buf || min > max`?

Reply:
Sorry for the late reply.
Yes, it should be `!buf || min > max`. Thanks to correct that.